src/Uno.UI/UI/Xaml/FrameworkPropertyMetadata.cs

```diff
@@ -13,9 +13,6 @@ namespace Microsoft.UI.Xaml
 	/// </remarks>
 	public class FrameworkPropertyMetadata : PropertyMetadata
 	{
-		private bool _isDefaultUpdateSourceTriggerSet;
-		private UpdateSourceTrigger _defaultUpdateSourceTrigger;
```

Comment:
This reduces the size of FrameworkPropertyMetadata. There is no need to store information that's really special only to TextBox.